UserShiftsApiService/UserShiftsApiService/Models/EmployeeItem.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace UserShiftsApiService.Models;
+
+public class EmployeeItem
+{
+    public string user_id { get; set; }
```

Comment:
use a C# convention car name like `UserId` and use an annotation on setting what `json` value this has. read up on it.